### PR TITLE
Workflow Logs GA: Release note and doc updates

### PR DESCRIPTION
**Pull request by rachelwhitton**
Workflow Logs will launch into general access by 7/1 

This PR includes: 

* Minor copy adjustments to /workflow-logs doc, removing "early access" context 
* A new release note announcing general access

### CODE REVIEW

File: source/releasenotes/2024-07-01-workflow-logs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+We’re excited to announce the launch of Workflow Logs, a powerful new feature enhancing visibility and streamlining troubleshooting operations on the Pantheon platform. Accessible directly from your site dashboard, next to the Support tab located in the top right corner, Workflow Logs provide critical insights for effectively managing your workflows.
+
+## Key features:
+* **Status overview**: View key information on the status of workflows, including progress statuses, along with additional details such as start time, end time, initiating team member, and environment.
+* **In-depth details**: Access code sync, error details, build logs, and Quicksilver posted messages from the expandable sections.
+* **Filtering**: The workflows displayed comprise a curated list of user-relevant workflows. Further narrow your focus by utilizing filters to quickly find specific workflows by environment or completion status.
+* **Archiving**: Export all visible workflows to a CSV file for archival purposes with the 'Download to CSV' button.
+
+<!-- TODO: Add image from gdoc draft -->
```
Comment (rachelwhitton) [inline source/releasenotes/2024-07-01-workflow-logs.md:15]: I need the image to add here, I asked Serena for this internally

File: source/releasenotes/2024-07-01-workflow-logs.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Our team will continuously enhance Workflow Logs through a planned list of improvements and ongoing feedback collection.
```
Comment (rachelwhitton) [inline source/releasenotes/2024-07-01-workflow-logs.md:19]: In the EA release note we specified that feedback would be collected via the EA program. How will feedback be collected in GA?

Comment (moreCoffie) [reply source/releasenotes/2024-07-01-workflow-logs.md:19]: What's the standard treatment for for updates on features in general access? Do we always include a note about feedback?

Comment (rachelwhitton) [reply source/releasenotes/2024-07-01-workflow-logs.md:19]: @moreCoffie My preference is not to mention feedback collection unless we're actively seeking and collecting feedback from a given feature with a defined process (e.g., have your customer fill out this form, or email that person).

**Comment by moreCoffie** [comment]
Can we push back merging this PR? Schedule it for July 3rd rather than July 1st.

**Comment by stevector** [comment]
That's fine. I updated the date in the substance of the PR

File: source/releasenotes/2024-07-01-workflow-logs.md (hunk 2 above)
Comment (moreCoffie) [reply source/releasenotes/2024-07-01-workflow-logs.md]: Yes, I prefer that too.

**Comment by moreCoffie** [comment]
I'm so sorry to have to ask again. There's a delay on wrapping up the engineering work before GA. I don't have enough clarity on the timeline yet. Can we un-schedule the merge for now?

**Review by stevector — CHANGES_REQUESTED** [review]
We'll need an updated publishing date based on the latest comment from @moreCoffie

**Comment by stevector** [comment]
Thanks @moreCoffie! I just added a review that'll block merging until we have an updated date.

File: source/releasenotes/2024-07-09-workflow-logs.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+We’re excited to announce the launch of Workflow Logs, a powerful new feature enhancing visibility and streamlining troubleshooting operations on the Pantheon platform. Accessible directly from your site dashboard, next to the Support tab located in the top right corner, Workflow Logs provide critical insights for effectively managing your workflows.
+
+## Key features:
+* **Status overview**: View key information on the status of workflows, including progress statuses, along with additional details such as start time, end time, initiating team member, and environment.
+* **In-depth details**: Access code sync, error details, build logs, and Quicksilver posted messages from the expandable sections.
+* **Filtering**: The workflows displayed comprise a curated list of user-relevant workflows. Further narrow your focus by utilizing filters to quickly find specific workflows by environment or completion status.
+* **Archiving**: Export all visible workflows to a CSV file for archival purposes with the 'Download to CSV' button.
+
+<!-- TODO: Add image from gdoc draft -->
```
Comment (stevector) [inline source/releasenotes/2024-07-09-workflow-logs.md:15]: I recommend that the image go above the list of key features

Comment (rachelwhitton) [reply source/releasenotes/2024-07-09-workflow-logs.md]: Done

**Comment by rachelwhitton** [comment]
Image has been added to release note; continuing to hold for merging until signal from @moreCoffie

**Comment by moreCoffie** [comment]
Steve and Rachel, GA is back on track. Can we schedule the merge for this Wed, Jul 24th?

**Comment by stevector** [comment]
@moreCoffie yes, we can merge then!

**Review by stevector — APPROVED** [review]

**Comment by stevector** [comment]
@moreCoffie, has this change gone out to all users? I'm not seeing it on my own testing account?

**Comment by moreCoffie** [comment]
@stevector the GA change has just been pushed less than an hour ago - feel free to merge this PR whenever you can

**Pull request merged**